packit/distgit.py

```diff
@@ -179,7 +179,12 @@ def upload_to_lookaside_cache(self, archive_path: str) -> None:
         logger.info("uploading to the lookaside cache")
         f = FedPKG(self.fas_user, self.local_project.working_dir)
         f.init_ticket()
-        f.new_sources(sources=archive_path)
+        try:
+            f.new_sources(sources=archive_path)
+        except Exception as ex:
+            logger.error("Your Fedora kerberos ticket is NOT valid: %r", ex)
+            return False
```

Comment:
I don't feel like this is the preferred pythonic way: return bool. I would actually `raise` back.

Reply:
`raise` exception is back again.